src/MUIPlacesAutocomplete.jsx

```diff
@@ -182,6 +182,14 @@ export default class MUIPlacesAutocomplete extends React.Component {
   // function signature as:
   // onInputValueChange(inputValue: string, stateAndHelpers: object)
   onInputValueChange(inputValue) {
+    const { textFieldProps: { onChange } } = this.props
```

Comment:
should rename `onChange` to `onInputValueChange` to match downshift, since `onChange` has an implied callback of an `Event`, not a simple value.